includes/wc-user-functions.php

```diff
@@ -108,7 +108,7 @@ function wc_create_new_customer( $email, $username = '', $password = '' ) {
 		$customer_id = wp_insert_user( $new_customer_data );
 
 		if ( is_wp_error( $customer_id ) ) {
-			return new WP_Error( 'registration-error', '<strong>' . __( 'ERROR', 'woocommerce' ) . '</strong>: ' . __( 'Couldn&#8217;t register you&hellip; please contact us if you continue to have problems.', 'woocommerce' ) );
+			return new WP_Error( 'registration-error', '<strong>' . __( 'Error', 'woocommerce' ) . ':</strong> ' . __( 'Couldn&#8217;t register you&hellip; please contact us if you continue to have problems.', 'woocommerce' ) );
```

Comment:
@claudiosmweb 
I was trying to merge similar WooCommerce strings. As you can see [here](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt-br/default?filters%5Bterm%5D=error&filters%5Buser_login%5D=&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filter=Filter&sort%5Bby%5D=original&sort%5Bhow%5D=asc):

* `Error` - 5 strings
* `ERROR` - 1 string

This change merges the two strings.

I will review all the "error messages" in a separate PR.